backend/spec/features/admin/orders/order_details_spec.rb

```diff
@@ -111,11 +108,14 @@
         expect(page).to have_content("Default")
       end
 
-      it "will show the variant sku" do
-        order = create(:completed_order_with_totals)
-        visit spree.edit_admin_order_path(order)
-        sku = order.line_items.first.variant.sku
-        expect(page).to have_content("SKU: #{sku}")
+      context "with a completed order" do
+        let!(:order) { create(:completed_order_with_totals) }
```

Comment:
Do you mean it's always preferred in `solidus` project(s) or it's generally always preferred? Because I prefer not to use it and I can find some good arguments why it should be avoided:
https://robots.thoughtbot.com/lets-not
http://linduxed.com/blog/2014/08/24/writing-more-readable-rspec-tests/

Of course there are always arguments pro and contra. Just asking out of curiosity.